src/twisted/internet/unix.py

```diff
@@ -52,7 +52,11 @@ def _ancillaryDescriptor(fd):
 
 
 @implementer(interfaces.IUNIXTransport)
-class _SendmsgMixin(object):
+class _SendmsgMixin(object):  # type: ignore[misc]
```

Comment:
I agree with your fix, and have implemented it.
_SendmsgMixin should not be marked as implementing IUNIXTransport if it does not fully implement it.

In twisted/internet/interfaces.py , maybe `IUNIXTransport` should have been a standalone interface, and not have inherited from `ITransport`.

Ideally, interfaces should be as minimal as possible, and avoiding inheritance hierarchies in interfaces is a good idea IMHO.

However, as much as possible, I want to avoid changing existing interfaces in Twisted.

Reply:
I shifted that change over to https://github.com/twisted/twisted/pull/1299